docs/repositories/commits.md

```diff
@@ -43,11 +44,17 @@ This area displays the overview of the code quality metrics for the {{ page.meta
     -   **Coverage variation:** Variation of code coverage relative to the target branch
     -   **Diff coverage:** Code coverage of the lines added or changed by the pull request
 {% endif %}
--   The **colors** depend on the [quality gate rules](../repositories-configure/adjusting-quality-settings.md) that are currently configured on your repository quality settings:
+
+    Depending on the languages being analyzed or if you haven't [set up coverage for your repository](../coverage-reporter/index.md), some metrics **may not be calculated** (represented by `-`).
+
+-   The **colors** depend on the [quality gate rules](../repositories-configure/adjusting-quality-settings.md) that are configured on your repository quality settings:
+
     -   **Green:** The metric passes the quality gate
     -   **Red:** The metric fails the quality gate
     -   **Gray:** There aren't quality gate rules configured for the metric
--   Depending on the languages being analyzed or if you haven't [set up coverage for your repository](../coverage-reporter/index.md), **some metrics may not be calculated** (represented by `-`).
+
+    !!! notes
+        If you make change the quality gate rules you must reanalyze the {{ page.meta.page_name }} to update the color of the **Issues**, while the color of the remaining metrics update as soon as you save the quality gate rules.
```

Comment:
Actually after doing a bit more investigation, it seems that only Coverage is different. So to update Issues, Complexity and Duplication. So maybe we can identify as coverage being different